hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestBlockStoragePolicy.java

```diff
@@ -1337,6 +1337,50 @@ public void testChooseSsdOverDisk() throws Exception {
     Assert.assertEquals(StorageType.DISK, targets[1].getStorageType());
   }
 
+  @Test
```

Comment:
Add some javadoc here will be more friendly here.

Reply:
Done.